docs/notebooks/model-levels.ipynb

```diff
@@ -0,0 +1,2052 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.        c=contrail["tau_contrail"],
</pre><p>Remove this commented line?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/contrailcirrus/pycontrails/pull/148/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Thank you!

4377b6e67178079f6262753d7226d695e0829883